src/library/core.ts

```diff
@@ -32,7 +36,10 @@ export default class Core {
     try {
       const command = commands.get(commandName);
       if (command) {
-        return command.execute(args, msg, this.client);
+        return command.execute(args, msg, {
+          client: this.client,
+          commands
+        });

```

Comment:
Working towards optional parameters. This is the only place that passes the data down, but not all commands utilize all the parameters, so I'm working to get these into an object and have the interface allow them all to be optional for the commands that are implementing it.

An issue related to this has been opened at: #109